util/util_windows.go

```diff
@@ -24,6 +24,11 @@ import (
 
 /* misc */
 
+// GetFdSoftLimit returns a current value for file descriptors soft limit.
+func GetFdSoftLimit() (uint64, error) {
+	return 0, nil
```

Comment:
Is there any consequence for windows node runners here?

Reply:
no, since set is noop